handlers/server.go

```diff
@@ -0,0 +1,188 @@
+package handlers
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"strconv"
+	"strings"
+
+	"github.com/gorilla/mux"
```

Comment:
can you use httprouter just to remain consistent with the engine?

Reply:
https://github.com/battlesnakeio/roadmap/issues/246